packages/components/src/components/ViewerMenuBar/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState } from "react";
```

Comment:
This file was mostly just renamed from ``packages/components/src/components/SquiggleOutputViewer/index.tsx``. I should have checked that commit closer to make sure it understood this was a rename, but it seems too late now without some significant annoyance.